platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/MapboxTest.java

```diff
@@ -68,17 +68,6 @@ public void testBlaBlaToken() {
     assertFalse(Mapbox.isAccessTokenValid("blabla"));
   }
 
-  @Test
-  public void testConnected() {
-    injectMapboxSingleton("pk.0000000001");
-
-    // test manual connectivity
-    Mapbox.setConnected(true);
-    assertTrue(Mapbox.isConnected());
-    Mapbox.setConnected(false);
-    assertFalse(Mapbox.isConnected());
-  }
-
```

Comment:
is this test no longer valid? can we come up with a replacement (if needed an instrumented test)?

Reply:
We already do have an instrumented test in place :)